ts/state/ducks/SessionTheme.tsx

```diff
@@ -138,6 +144,23 @@ export const switchHtmlToDarkTheme = () => {
   document.documentElement.style.setProperty('--color-button-green', darkButtonGreen);
   document.documentElement.style.setProperty('--color-modal-background', darkModalBackground);
   document.documentElement.style.setProperty('--border-session', darkColorSessionBorder);
+
+  document.documentElement.style.setProperty(
+    '--color-request-banner-background',
+    darkMessageRequestBannerBackground
+  );
+  document.documentElement.style.setProperty(
+    '--color-request-banner-icon-background',
+    darkMessageRequestBannerIconBackground
+  );
+  document.documentElement.style.setProperty(
+    '--color-request-banner-unread-background',
+    darkMessageRequestBannerUnreadBackground
+  );
+  document.documentElement.style.setProperty(
+    '--color-request-banner-icon',
+    darkMessageRequestBannerIcon
+  );
```

Comment:
are those really new colors ?
If not, be sure to rely on existing values instead. we don't want to add too many colors to the body element